src/lib/components/organizationUsageLimits.svelte

```
@@ -0,0 +1,396 @@
+    import type { Models } from '@appwrite.io/console';
+
+    // Props
+    let {
+        organization,
+        projects = [],
+        members = [],
+        storageUsage = 0
+    }: {
+        organization: Organization;
+        projects?: Models.Project[];
+        members?: any[];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)